lib/components/Blocks/Header/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { FlexBox } from "@/components/Layout/_FlexBox"
+import { Avatar, AvatarFallback, AvatarImage } from "@/ui/avatar"
+import { forwardRef } from "react"
+
+interface HeaderProps {
+  title: string
+  subtitle: string
+  src: string
+  alt: string
+}
+
+export const Header = forwardRef<HTMLDivElement, HeaderProps>(
+  ({ title, subtitle, src, alt }, ref) => {
+    return (
+      <FlexBox ref={ref} className="px-10 py-6">
+        <Avatar size="xlarge">
```

Comment:
I think we might want to make the `Avatar` API more strict so it doesn't allow so much composability. I think these two things could be passed as props. The internals (shadcn implementation) could stay nicely decoupled, but we could expose another component that's more strict, perhaps?

That's what we do with `Button`, in case you need an example.